lumen/ai/llm.py

```diff
@@ -29,6 +29,7 @@ class Llm(param.Parameterized):
     # Allows defining a dictionary of default models.
     model_kwargs = param.Dict(default={})
 
+    _supports_stream = True
```

Comment:
I'm a tad confused about this vs supports_model_stream.

Reply:
Correct, not convinced we actually need both.

Reply:
Yes, I believe Mistral doesn't have first class support in instructor.
https://github.com/holoviz/lumen/pull/697/files#diff-dc864a2206353ab4b7498c9a9361f5c45af0c5fd84adbc59fe054515346e55d4R290